lib/Twig/Cache/Filesystem.php

```
@@ -44,17 +44,9 @@ public function generateKey($name, $className)
     /**
      * {@inheritdoc}
      */
-    public function has($key)
-    {
-        return is_file($key);
-    }
-
-    /**
-     * {@inheritdoc}
-     */
     public function load($key)
     {
-        require_once $key;
+        @include_once $key;
```

silencing errors is a bad idea IMO